client/src/game/reducers/initialStates/initialState.ts

```diff
@@ -25,7 +38,10 @@ export default function initialState(metadata: GameMetadata): State {
     datetimeFinished: null,
 
     metadata,
-    cardIdentities: [],
+    cardIdentities: initArray(startingDeckSize, {
+      suitIndex: null,
+      rank: null,
+    }),
```

Comment:
1. `startingDeckSize` returns the size of the deck on turn 1. Do you mean to use the size of the deck before cards are dealt?

2. Now everywhere where we checked for undefined before we need to check for {null, null}, I can update that.

3. If this is actually a good idea then there should be similar code in `loadGameJSON.ts`

Reply:
1) yes
2) yes
3) yes

Reply:
fixed